src/scene/graphics/quad-render-utils.js

```diff
@@ -100,7 +100,7 @@ function drawTexture(device, texture, target, shader, rect, scissorRect) {
     const useBlend = arguments[6];
     Debug.call(() => {
         if (useBlend !== undefined) {
-            Debug.warnOnce('pc.drawTexture no longer accepts useBlend parameter, and blending state needs to be set up using GraphicsDevice.setBlendState.');
+            Debug.warnOnce('pc.drawTexture no longer accepts useBlend parameter, and blending state needs to be set up using WebglGraphicsDevice.setBlendState.');
```

Comment:
and this needs to stay as GraphicsDevice too

Reply:
Fixed.